cordapp/src/main/kotlin/net/corda/plugins/CordappPlugin.kt

```diff
@@ -57,11 +64,12 @@ class CordappPlugin : Plugin<Project> {
         val directDeps = runtimeConfiguration.allDependencies - excludeDeps
         // We want to filter out anything Corda related or provided by Corda, like kotlin-stdlib and quasar
         val filteredDeps = directDeps.filter { dep ->
-            excludes.none { exclude -> (exclude["group"] == dep.group) && (exclude["name"] == dep.name) }
+            excludes.none { exclude -> (exclude["name"] == dep.name) && (
+                    (exclude["group"] == dep.group) || dep.group?.startsWith("com.github.corda.") ?: false) }
```

Comment:
Could you instead scan for the "com.github.corda" packages and add them to the "exclude deps" so we can keep the excluded deps in a single structure.

Reply:
Actually, looking more closely at the code:
```kotlin
val excludeDeps = project.configuration("cordapp").allDependencies +
        project.configuration("cordaCompile").allDependencies +
        project.configuration("cordaRuntime").allDependencies
val directDeps = runtimeConfiguration.allDependencies - excludeDeps
// We want to filter out anything Corda related or provided by Corda, like kotlin-stdlib and quasar
val filteredDeps = directDeps.filter { dep ->
    excludes.none { exclude -> (exclude["group"] == dep.group) && (exclude["name"] == dep.name) }
}
```
You appear to be talking about `excludeDeps`, whereas the `excludes` (which is where the bug is) are actually applied to `filteredDeps`. This PR is about broadening the filter to include jitpack versions of the `excludes` artifacts.